src/main/java/InvalidInput.java

```diff
@@ -0,0 +1,12 @@
+public class InvalidInput {
+    public static final String INVALID_INPUT = "    Invalid input. Please enter again\n";
+
+    /**
+     * returns a prompt to re-enter the input when an invalid input is entered.
+     *
+     * @return invalid input warning
+     */
+    public static String printInvalidInput() {
+        return INVALID_INPUT;
+    }
+}
```

Comment:
Is there a reason to put this possible method as a whole class on its own?

Reply:
Thank you for pointing this out! I have rearranged some similar methods into the same class to improve readability and future tracking.